<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyBoard</class>
 <widget class="QWidget" name="KeyBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="toolButton_x">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>213</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_q">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>113</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>q</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_7">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>63</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_esc">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>13</y>
     <width>61</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_0">
   <property name="geometry">
    <rect>
     <x>519</x>
     <y>63</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_o">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>113</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>o</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_k">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>163</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>k</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_y">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>113</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_w">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>113</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>w</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_xinghao">
   <property name="geometry">
    <rect>
     <x>564</x>
     <y>213</y>
     <width>70</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_dot">
   <property name="geometry">
    <rect>
     <x>589</x>
     <y>113</y>
     <width>51</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_c">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>213</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>c</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_b">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>213</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>b</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_i">
   <property name="geometry">
    <rect>
     <x>439</x>
     <y>113</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>i</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_line2">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>213</y>
     <width>61</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>\</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_u">
   <property name="geometry">
    <rect>
     <x>389</x>
     <y>113</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>u</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_e">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>113</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>e</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_tab">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>63</y>
     <width>51</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Tab</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_g">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>163</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>g</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_s">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>163</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_down">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>263</y>
     <width>70</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>down</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_space">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>263</y>
     <width>401</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Space</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_inv">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>13</y>
     <width>61</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Inv</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_6">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>63</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_m">
   <property name="geometry">
    <rect>
     <x>439</x>
     <y>213</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_a">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>163</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>a</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="display">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>13</y>
     <width>351</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_t">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>113</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>t</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_h">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>163</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_p">
   <property name="geometry">
    <rect>
     <x>539</x>
     <y>113</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>p</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_line1">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>163</y>
     <width>51</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_z">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>213</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>z</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_4">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>63</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_capslock">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>113</y>
     <width>71</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Caps</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_up">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>213</y>
     <width>70</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>up</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_enter">
   <property name="geometry">
    <rect>
     <x>523</x>
     <y>13</y>
     <width>111</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_f">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>163</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>f</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>63</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_n">
   <property name="geometry">
    <rect>
     <x>389</x>
     <y>213</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>n</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_fenhao">
   <property name="geometry">
    <rect>
     <x>569</x>
     <y>163</y>
     <width>71</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_left">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>263</y>
     <width>70</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>left</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_wenhao">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>163</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_r">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>113</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>r</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_l">
   <property name="geometry">
    <rect>
     <x>519</x>
     <y>163</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_8">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>63</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_1">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>63</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_5">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>63</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>63</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_right">
   <property name="geometry">
    <rect>
     <x>564</x>
     <y>263</y>
     <width>70</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>right</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_line3">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>213</y>
     <width>55</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>_</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_backspace">
   <property name="geometry">
    <rect>
     <x>569</x>
     <y>63</y>
     <width>71</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>DEL</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_9">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>63</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_d">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>163</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>d</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_v">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>213</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_j">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>163</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>j</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
